### PR TITLE
T-37 - Update UI

### PR DESCRIPTION
**Pull request by AntonBessarabov**
Changed layout
Updated all pages
Delete asset page moved to modal
Removed unnecessary scripts

> One more thing - After delete it would be good to see message - asset {AssetName} was deleted.

Message not implemented
Also display-names not changed

**Need to discuss the content of the main page**

<details>

<summary>Some screenshots. Click to see</summary>

default
default
default
default
default
default

</details>

### CODE REVIEW

**Comment by AntonBessarabov** [comment]
[T-37 Update UI](https://trello.com/c/kXx8YRbM/37-t-37-update-ui)

File: PortfolioKeeper/Models/ViewModels/RegisterViewModel.cs [hunk 1]
```diff
@@ -11,17 +11,17 @@ public class RegisterViewModel
 
         [Required]
         [StringLength(256, ErrorMessage = "The {0} must be at least {2} characters long.", MinimumLength = 3)]
-        [Display(Name = "UserName")]
+        [Display(Name = "User Name")]
```
Comment (goremukin) [inline PortfolioKeeper/Models/ViewModels/RegisterViewModel.cs]: User name

File: PortfolioKeeper/Models/ViewModels/RegisterViewModel.cs [hunk 2]
```diff
@@ -11,17 +11,17 @@ public class RegisterViewModel
...
         public string UserName { get; set; }
 
         [Required]
         [StringLength(50, ErrorMessage = "The {0} must be less than {1} characters long.")]
-        [Display(Name = "FirstName")]
+        [Display(Name = "First Name")]
```
Comment (goremukin) [inline PortfolioKeeper/Models/ViewModels/RegisterViewModel.cs]: First name

File: PortfolioKeeper/Models/ViewModels/RegisterViewModel.cs [hunk 3]
```diff
@@ -11,17 +11,17 @@ public class RegisterViewModel
...
         public string FirstName { get; set; }
 
         [Required]
         [StringLength(50, ErrorMessage = "The {0} must be less than {1} characters long.")]
-        [Display(Name = "LastName")]
+        [Display(Name = "Last Name")]
```
Comment (goremukin) [inline PortfolioKeeper/Models/ViewModels/RegisterViewModel.cs]: Last name

File: PortfolioKeeper/Views/Asset/AssetTypeChildrenSearch.cshtml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ﻿@model IEnumerable<PortfolioKeeper.Models.ViewModels.AssetTypeViewModel>
 
-<label>Children Asset Type</label><br />
+<label>Asset Subtype</label>
```
Comment (goremukin) [inline PortfolioKeeper/Views/Asset/AssetTypeChildrenSearch.cshtml]: Asset subtype

File: PortfolioKeeper/Views/Manage/ChangePassword.cshtml [hunk 5]
```diff
@@ -3,38 +3,50 @@
...
-    <div class="form-group">
-        @Html.LabelFor(m => m.ConfirmPassword, new { @class = "col-md-2 control-label" })
-        <div class="col-md-10">
-            @Html.PasswordFor(m => m.ConfirmPassword, new { @class = "form-control" })
+<div class="row justify-content-center">
+    <div class="col-12 col-md-8 col-lg-6">
+        <div class="card">
+            <div class="card-header text-center">
+                <h2 class="card-description">
+                    Change Password
```
Comment (goremukin) [inline PortfolioKeeper/Views/Manage/ChangePassword.cshtml]: Change password

**Comment by AntonBessarabov** [comment]
Main page update

<details>

<summary>Screenshot. Click to see</summary>

default

</details>

File: PortfolioKeeper/Content/Montserrat.css [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+}
+/* cyrillic */
+@font-face {
+  font-family: 'Montserrat';
+  font-style: normal;
+  font-weight: 200;
+  src: local('Montserrat ExtraLight'), local('Montserrat-ExtraLight'), url(https://fonts.gstatic.com/s/montserrat/v12/JTURjIg1_i6t8kCHKm45_aZA3g3D_u50.woff2) format('woff2');
+  unicode-range: U+0400-045F, U+0490-0491, U+04B0-04B1, U+2116;
+}
+/* vietnamese */
```
Comment (stas-litvinenka) [inline PortfolioKeeper/Content/Montserrat.css]: can we remove this?

File: PortfolioKeeper/Views/Shared/_Layout.cshtml [hunk 7]
```diff
@@ -2,43 +2,72 @@
...
-                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
-                    <span class="icon-bar"></span>
-                    <span class="icon-bar"></span>
-                    <span class="icon-bar"></span>
-                </button>
-                @Html.ActionLink("Application name", "Index", "Home", new { area = "" }, new { @class = "navbar-brand" })
+<body class="">
+    <div class="wrapper">
+        <div class="sidebar" data-color="blue">
+            @*data-color="blue | green | orange | red | yellow"*@
```
Comment (stas-litvinenka) [inline PortfolioKeeper/Views/Shared/_Layout.cshtml]: remove?

File: PortfolioKeeper/Content/mainPage.css [hunk 8]
```diff
@@ -0,0 +1,54 @@
+﻿body {
+    background: url('Img/squirrel-wallpaper-1920x1080-003.jpg');
```
Comment (EdwOK) [inline PortfolioKeeper/Content/mainPage.css]: 4 in 1: https://developer.mozilla.org/ru/docs/Web/CSS/background
`background: url('Img/squirrel-wallpaper-1920x1080-003.jpg') no-repeat center fixed`

File: PortfolioKeeper/Content/mainPage.css [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-position: center;
+}
+
+.main-panel {
+    background: none;
```
Comment (EdwOK) [inline PortfolioKeeper/Content/mainPage.css]: okay, what about this - `display: none`?

Comment (AntonBessarabov) [reply PortfolioKeeper/Content/mainPage.css]: Also disables the display of nested items

**Pull request merged**